### ISSUE TITLE
add more AttributeNames

### ISSUE THREAD

**Issue by ajkiosd** [Raizlabs/BonMot#256]
hi
is it possible to add `NSStrokeColorAttributeName`  `NSStrokeWidthAttributeName`  `NSShadowAttributeName` to  StringStyle?

**Comment by ZevEisenberg** [comment]
@ajkiosd yes, it's possible to add all of those! If you'd like to submit a pull request, I believe that an implementation of `NSStrokeColorAttributeName` and `NSStrokeWidthAttributeName` would look a lot like the current implementation of underline and strikethrough, so those would make for a good example.

It looks like `NSShadowAttributeName` needs to be paired with an `NSShadow` object. Does this work on all four supported platforms? If not, it will need to be guarded per-platform, and there is ample reference for how to do this in the BonMot code already. An entry in the example app that shows off some of these effects would also be nice, but we can get a designer to work on that once the feature is built.

As a workaround, since those features are not currently supported, you can use `StringStyle.extraAttributes` or `StringStyle.Part.extraAttributes` to pass a dictionary of attributes wherever you are already using BonMot.
